### PR TITLE
Unit Types!

### PR DESCRIPTION
**Pull request by rmjohnson**

### CODE REVIEW

File: Assets/Source/Entities/Behaviors/Breed.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	public void StopBirthing()
+	{
+		CancelInvoke("GiveBirth");
+		IsBirthing = false;
+	}
+
+	private void GiveBirth()
+	{
+		Instantiate(BaseUnitGameObject, birthingPosition, birthingRotation);
```
Comment (leyarotheconquerer) [inline Assets/Source/Entities/Behaviors/Breed.cs:31]: Possibly add a random offset from the breeder? Maybe like this:

``` diff
        public bool IsBirthing;
+       public float BirthRadius = 10f;
        public Vector3 birthingPosition = new Vector3();
        public Quaternion birthingRotation = new Quaternion();
```

``` diff
        private void GiveBirth()
        {
+               birthingPosition = new Vector3(Random.value, 0, Random.value);
+               birthingPosition.Normalize();
+               birthingPosition = birthingPosition * BirthRadius + transform.position;
+
                Instantiate(BaseUnitGameObject, birthingPosition, birthingRotation);
```

File: Assets/Source/Entities/Behaviors/Harvester.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Harvester : MonoBehaviour
+{
+	// Stats
+	public float HarvestingRate;
+	public float HarvestRange;
+	public float DepositRange;
+	public float DeposityRetryRate;
```
Comment (leyarotheconquerer) [inline Assets/Source/Entities/Behaviors/Harvester.cs]: Hmm... DeposityRetryRate?

Did you mean DepositRetryRate? Or is this a feature?

Comment (leyarotheconquerer) [inline Assets/Source/Entities/Behaviors/Harvester.cs]: Nice 'y'.

**Comment by leyarotheconquerer** [comment]
I noticed a few things. They should be in the comments above. Otherwise, it looks good.

As far as I'm concerned, it's ready for merging.

File: Assets/Source/Entities/Behaviors/Harvester.cs (hunk 2 above)
Comment (rmjohnson) [inline Assets/Source/Entities/Behaviors/Harvester.cs]: =(

**Comment by rmjohnson** [comment]
Fixed! =D

File: Assets/Source/Entities/Behaviors/Harvester.cs (hunk 2 above)
Comment (rmjohnson) [inline Assets/Source/Entities/Behaviors/Harvester.cs]: #featurenotabug

**Pull request merged**